samza-core/src/main/java/org/apache/samza/task/StreamOperatorTask.java

```diff
@@ -39,60 +37,60 @@
 
 /**
  * A {@link StreamTask} implementation that brings all the operator API implementation components together and
- * feeds the input messages into the user-defined transformation chains in {@link StreamApplication}.
+ * feeds the input messages into the user-defined transformation chains in {@link org.apache.samza.operators.OperatorSpecGraph}.
```

Comment:
Minor: {@link OperatorSpecGraph} everywhere instead of fully qualified name.

Reply:
Good catch!